core/src/apps/webauthn/fido2.py

```diff
@@ -1164,6 +1203,7 @@ def msg_register_sign(challenge: bytes, cred: U2fCredential) -> bytes:
 
 
 def msg_authenticate(req: Msg, dialog_mgr: DialogManager) -> Cmd:
+    global _last_good_auth_check_cid
```

Comment:
move this below, just before the assignment?

Reply:
Done. Rebased on top of andrewkozlik/733.